src/main/java/com/Futurecollars/Lekcja3/Zadanie5/PithagorasThesis.java

```diff
@@ -0,0 +1,20 @@
+package com.Futurecollars.Lekcja3.Zadanie5;
+
+public class PithagorasThesis {
+    public static int a;
+    public static int b;
+    public static int c;
+
+    public static int calculation()
```

Comment:
dodatkowo czy nazwa metody calculation tłumaczy co ta metoda robi? kompletnie nie :) może lepiej ją nazwać isPithagorasTheoremFullfilled i metoda będzie ci zamiast jakiegoś tekstu zwracała true albo false w zależności czy spełnia czy nie spełnia twierdzenie pitagorasa. Oczywiście można to jeszcze rozwiązać na inne sposoby. Pomyśl jak to lepiej rozwiązać